e2e-polybft/property/property_test.go

```diff
@@ -55,3 +56,74 @@ func TestProperty_DifferentVotingPower(t *testing.T) {
 		require.NoError(t, cluster.WaitForBlock(numBlocks, blockTime*time.Duration(numBlocks)))
 	})
 }
+
+func TestProperty_DropValidators(t *testing.T) {
+	t.Parallel()
+
+	const (
+		blockTime = time.Second * 4
```

Comment:
Should we pass this as a parameter to cluster callback option (`framework.WithBlockTime`), because otherwise it is 2s by default.